apps/web/pages/apps/installed.tsx

```diff
@@ -507,7 +507,7 @@ function ConnectOrDisconnectIntegrationButton(props: {
     );
   }
   /** We don't need to "Connect", just show that it's installed */
-  if (["daily_video", "huddle01_video", "jitsi_video"].includes(props.type)) {
```

Comment:
This caused the Daily.co integration to display a "Connect" button which is the incorrect behaviour. Fixed by setting an `isGlobal` boolean in the metadata.

Reply:
Thanks for the update and the heads-up @zomars 👍